app/views/LocationTracking.js

```diff
@@ -288,7 +294,9 @@ class LocationTracking extends Component {
   }
 
   overlap() {
-    this.props.navigation.navigate('OverlapScreen', {});
+    if (this.state.isUserOffline) {
+      alert(languages.t('offlineMessage'));
```

Comment:
Add a test

Reply:
add a test means unit test case?

Reply:
Yes please. LocationTracking.spec.js and also for Settings